schema/config-linux.json

```diff
@@ -239,7 +242,9 @@
                             },
                             "swappiness": {
                                 "id": "https://opencontainers.org/schema/bundle/linux/resources/memory/swappiness",
-                                "$ref": "defs.json#/definitions/uint64Pointer"
+                                "type": "integer",
+                                "minimum": 0,
+                                "maximum": 100
```

Comment:
On Thu, Aug 18, 2016 at 02:41:21AM -0700, Aleksa Sarai wrote:
> > @@ -239,7 +242,9 @@
> >                              },
> >                              "swappiness": {
> >                                  "id": "https://opencontainers.org/schema/bundle/linux/resources/memory/swappiness",
> > -                                "$ref": "defs.json#/definitions/uint64Pointer"
> > +                                "type": "integer",
> > +                                "minimum": 0,
> > +                                "maximum": 100
> 
> The minimum is `-1` for `swappiness`.

The Go type is unsigned, so we're blocking -1 there.  More discussion
on this issue, whether we need to allow -1, and potential alternatives
in #499.